controller.php

```diff
@@ -108,6 +108,10 @@ protected function registerAssets()
             array('javascript', 'jquery'),
             array('javascript', 'mesch/alert'),
         ));
+

```

Comment:
you shouldn't do this, please have a look a few lines up. In 5.7 we register the assets and then require them in the controller. That way we can also ensure that we only include them if needed. With the code you've added they will be included in every page, no matter if there's a form in it or not.